uaclient/tests/test_apt.py

```diff
@@ -977,10 +976,71 @@ def test_setup_apt_proxy_config(
         self,
         m_util_remove_file,
         m_util_write_file,
-        kwargs,
+        args,
         expected_remove_calls,
         expected_write_calls,
     ):
-        setup_apt_proxy(**kwargs)
+        setup_apt_proxy(*args)
         assert expected_remove_calls == m_util_remove_file.call_args_list
         assert expected_write_calls == m_util_write_file.call_args_list
+
+    @pytest.mark.parametrize(
+        "subp_return_value, expected_ret",
+        [
+            (("", ""), None),
+            (("nonsense", ""), None),
+            (("", "nonsense"), None),
+            (("key='value'", ""), "value"),
+        ],
+    )
+    @pytest.mark.parametrize(
+        "protocol, expected_apt_key",
+        [("http", "Acquire::http::Proxy"), ("https", "Acquire::https::Proxy")],
+    )
+    @mock.patch("uaclient.util.subp")
+    def test_get_current_proxy_value(
```

Comment:
I think we can also have that test for the snap and livepatch implementations

Reply:
there should be `test_get_snap_config_option_value` and `test_get_livepatch_config_option_value` in `test_livepatch.py` unless you're referring to something else?

Reply:
My mistake here, I completely missed those tests